docs/application/native/guides/device/system.md

```diff
@@ -644,6 +644,7 @@ The following table lists the speech feature keys:
 | Key                                      | Type   | Description                              |
 |------------------------------------------|--------|------------------------------------------|
 | `http://tizen.org/feature/speech.control` | `bool` | The platform returns `true` for this key, if the device supports voice control. |
+| `http://tizen.org/feature/speech.control_manager` | `bool` | The platform returns `true` for this key, if the device supports voice control manager. |
```

Comment:
@sungraejo Sorry for the late check. The dotnet/guides/device/system.md is not listing feature keys in the way native/guides/device/system.md is doing.
Also it seems hard to change the system.md for dotnet as like system.md for native.

I'm now considering opening sentence of the Feature Keys section,
_The static feature keys are identical to the ones used in Tizen native applications. For more information, see the native System Information guide._

@sungraejo Would you please confirm, that this added feature key is also identically applied to Native? 
If adding this feature is same to native also, I think we can add the feature key to the native guide as you did, instead of updating dotnet/guide.

Sorry for the inconvenience about it.